src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Shape {
+    private String color;
+    private float radius;
+
+    public Circle(String color, float radius) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public float getRadius() {
+        return radius;
+    }
+
+    @Override
+    public float area() {
+        return (float) (3.14 * getRadius() * getRadius());
+    }
+
+    @Override
+    public void drawShape() {
+        System.out.println("Drawing "
+                + getColor()
+                + " circle with radius equals to "
+                + getRadius()

```

Comment:
```suggestion
                + radius 
```
you have direct access  to this field

fix all comments from this class and fix other classes accordingly